### PR TITLE
Add more endpoints

### PR DESCRIPTION
**Pull request by quietshu**
~~**Don't merge**: this PR depends on https://github.com/zeit/now-examples/pull/271.~~

Add 2 more endpoints:
- `/list-all.json`: get the [detailed list of all examples](https://github.com/zeit/now-examples/pull/271/files) in zeit/now-examples
- `/info.json?repo=<repository_url>`: get the basic info of a github repository

All existing APIs are not affected.

### CODE REVIEW

File: endpoints/info.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const D = decodeURIComponent
+
+module.exports = async (req, res) => {
+  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader('Access-Control-Allow-Methods', 'GET')
+  res.setHeader('Cache-Control', 'max-age=300, stale-while-revalidate=300')
+  
+  if (req.method !== 'GET') {
+    send(res, 501, {
```
Comment (joecohens) [inline endpoints/info.js]: @quietshu `micro.send` or `const {send} = require('micro')`

File: lib/example-list.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      }
+    }
+  )
+
+  if (response.status !== 200) {
+    log(`Non-200 response code from GitHub: ${response.status}`)
+    return null
+  }
+
+  const parsed = await response.json()
```
Comment (joecohens) [inline lib/example-list.js]: Maybe we want to check if the json has content here: 

```
  if (parsed.length === 0) {
    log(`Invalid response code from GitHub`);
    return null;
  }
```

File: lib/repo-info.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      }
+    }
+  )
+
+  if (response.status !== 200) {
+    log(`Non-200 response code from GitHub: ${response.status}`)
+    return null
+  }
+
+  const parsed = await response.json()
```
Comment (joecohens) [inline lib/repo-info.js]: Same here

File: lib/example-list.js (hunk 2 above)
Comment (quietshu) [reply lib/example-list.js]: Not sure about this, already checked `response.status` 🤔 and why `length`? Did you mean `size`?

Comment (joecohens) [reply lib/example-list.js]: Oh yep sorry I just tried the endpoint and it's not an array

**Pull request merged**